pkg/acsengine/defaults.go

```diff
@@ -98,8 +98,10 @@ func setDefaultCerts(a *api.Properties) (bool, error) {
 
 	ips := []net.IP{firstMasterIP}
 
-	for i := 1; i < a.MasterProfile.Count; i++ {
-		ips = append(ips, net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)})
+	// Include the Internal load balancer as well
+	for i := 1; i < (a.MasterProfile.Count + 1); i++ {
+		ip := net.IP{firstMasterIP[12], firstMasterIP[13], firstMasterIP[14], firstMasterIP[15] + byte(i)}
```

Comment:
I'm still stuck on what I presume is my own confusion...  `net.IP` is a type alias for `[]byte`, yeah?

It SEEMS like the code here would be doing the following. At least, from eye-balling it, since it seems to work:

```
ip[0] = firstMasterIP[12]
ip[1] = firstMasterIP[13]
ip[2] = firstMasterIP[14]
ip[3] = firstMasterIP[15] + byte(i)
```

Since you're using the array literal initializer syntax.